core/src/main/javascript/pt/webdetails/cgg/resources/cgg/interop.js

```diff
@@ -10,11 +10,9 @@
 * basis, WITHOUT WARRANTY OF ANY KIND, either express or  implied. Please refer to
 * the license for the specific language governing your rights and limitations.
 */
-define([
-    './util'
-], function(util) {
+define(function() {
 
-    var javaLang = util.global.java.lang;
+    var javaLang = java.lang;
```

Comment:
is `java` a global variable or did you miss `globalThis` on this one?

Reply:
Yes, `java` is a global variable. And I've relaxed using `globalThis` everywhere.